linera-execution/src/system.rs

```diff
@@ -375,7 +373,7 @@ where
         use SystemOperation::*;
         let mut outcome = RawExecutionOutcome {
             authenticated_signer: context.authenticated_signer,
-            refund_grant_to: context.refund_grant_to(),
+            refund_grant_to: Some(context.refund_grant_to()),
```

Comment:
No. We should not force refunds.

Reply:
Ah, you're right. We actually had two `Option`s previously here - the outer one (return type) and inner one (`Some(Account { owner: None })`) and previously `Some(None)` would not refund at all.